cryptol-remote-api/python/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Revision history for `cryptol` Python package
 
-## 2.12.2 -- YYYY-MM-DD
+## 2.12.4 -- YYYY-MM-DD
+
+* NEW CHANGELOG ENTRIES SINCE 2.12.2 GO HERE
+
+## 2.12.2 -- 2021-12-21
```

Comment:
Let's make an explicit `2.12.2 release` commit, _then_ bump things past it

Reply:
Good suggestion! This is now ded73dc